autoload/bazel/workspace.vim

```diff
@@ -0,0 +1,51 @@
+""
+" @section Introduction, intro
+" @library
+" Implements common path functionality used by bazel-related plugins.  For
+" more details about Build file structure/terminology, see:
+" https://bazel.build/versions/master/docs/build-ref.html
+
+""
+" @private
+" Returns the root path for a Bazel workspace file (given the current working
+" directory). {path} is the absolute path.
+function! bazel#workspace#GetBazelRootForCwd() abort
```

Comment:
Could callers just pass the `getcwd()` to GetBazelRootPath themselves? Seems better for separation of concerns and having less "surface area" to library code.

Reply:
Sure, removed.